src/main/resources/db/migration/V2__create_table_employee.sql

```diff
@@ -0,0 +1,10 @@
+create table employee(
+    id UUID not null,
+    person_id UUID not null,
```

Comment:
No desenho esse campo era nullable, não?
Como esse campo vai depender da integração com o sistema da Pat, talvez faça sentido controlar a obrigatoriedade no código. Deixar `not null` vai te impedir de criar um employe se a integração ainda não estiver rolando.